robots.txt

```diff
@@ -1,21 +1,50 @@
 User-agent: *
-Disallow: /admin/
-Disallow: /accounts/
-Disallow: /problem/*/pdf
+Disallow: /500
```

Comment:
Minor nit: we should probably sort these so that checking if a URL is in this file is not O(n) for a human. `sort < robots.txt` should do that for you; you'd just need to move `User-agent` back up.